controllers/comments-controllers.js

```diff
@@ -17,6 +17,17 @@ exports.getComments = (req, res, next) => {
     .catch(next);
 };
 
+exports.removeComment = (req, res, next) => {
+    const { comment_id } = req.params
+    return deleteComment(comment_id)
+    .then((comment)=> {
+        if (comment.length === 0) {
```

Comment:
A comment is an object so doesn't have a length property 🤔

Reply:
sorry this is my code just being unnecessarily confusing haha,  comment is response.rows here so i'm just checking if its an empty array or not